pkg/cmd/roachtest/tests/failover.go

```diff
@@ -66,6 +75,97 @@ func registerFailover(r registry.Registry) {
 	}
 }
 
+func randSleep(ctx context.Context, rng *rand.Rand, max time.Duration) {
+	randTimer := time.After(randutil.RandDuration(rng, max))
+	// Randomly sleep up to the lease renewal interval, to vary the time
+	// between the last lease renewal and the failure. We start the timer
+	// before the range relocation above to run them concurrently.
```

Comment:
nit: This comment belongs at the call site, not in the function, and it's no longer accurate either.